client/src/components/AddButton.js

```diff
@@ -1,14 +1,14 @@
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { Context } from '../hooks/Store';
+import { Context } from '../Store';
```

Comment:
As you could potentially use multiple Context's you might consider renaming this one to convey which context this is about. Something like `WatchlistContext`maybe?

Reply:
Not yet planning to use more Context, but still you are right :)